batch_job/src/amazon_ec2.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/sh
```

Comment:
Since a large section of batch_job_amazon.c is devoted to creating this file, is it necessary to have it also included? Not sure, but it feels unneeded. If you do remove it I think we want to add it to the gitignore for cctools so it does not get inadvertently added later on.